src/textual/app.py

```diff
@@ -1259,7 +1266,8 @@ def switch_screen(self, screen: Screen | str) -> AwaitMount:
         return AwaitMount(self.screen, [])
 
     def install_screen(self, screen: Screen, name: str | None = None) -> AwaitMount:
-        """Install a screen.
+        """Install a screen. Installing a screen prevents Textual from destroying it when it
```

Comment:
Perhaps break this up into a couple of lines [to stick to the convention we've adopted to keep the first line a stand-alone description](https://github.com/Textualize/textual/wiki/Documentation-standards#function--method)?